main.go

```diff
@@ -227,7 +227,8 @@ func main() {
 	}
 
 	// Setup Schemes for SSO if template instance is available.
-	if argocd.IsTemplateAPIAvailable() {
+	if argocd.CanUseKeycloakWithTemplate() {
+		setupLog.Info("Keycloak instance can be managed using OpenShift Template")
```

Comment:
LGTM, one final, tiny request then good to go: can we also add an Info statement (same as above) that outputs for the case where OpenShift Template is not available? Basically the opposite case to what is in the if statement above. 

Logging something like "Keycloak instance cannot be managed, as DeploymentConfig API is not installed on the cluster."